integrated_channels/canvas/client.py

```diff
@@ -183,28 +236,33 @@ def delete_course_completion(self, user_id, payload):  # pylint: disable=unused-
         pass
 
     # Private Methods
-
-    def _update_course_details(self, course_id, serialized_data):
+    def _update_course_details(self, course_id, course_details):
         """
         Update a course for image_url (and possibly other settings in future)
-        This is used only for settings that are not settable in the initial course creation
+        Also sets course to 'offer' state by sending 'course[event]=offer'
```

Comment:
what does the offer state mean in this context? Does it have an effect on how we interact with the course?

Reply:
This could be resolved by adding a link to the relevant Canvas API.